lib/tinyset.rs

```diff
@@ -41,18 +47,26 @@ impl<'a, X:u64set::Fits64 + Ord + 'a> Iterator for Set64Iter<'a, X> {
     }
 }
 
-impl <'a, T: tinyset::Fits64 + Ord> tinyset_Set64<T> {
-    pub fn iter(&'a self) -> Set64Iter<'a, T> {
+impl <T: tinyset::Fits64 + Ord> tinyset_Set64<T> {
+    pub fn iter(&self) -> Set64Iter<T> {
         Set64Iter::new(self)
     }
 }
 
+impl <'a, T: tinyset::Fits64 + Ord> tinyset_Set64<T> {
+    /* In cases when order really, definitely, 100% does not matter,
+     * a more efficient iterator can be used.
+     */
+    pub fn unsorted_iter(&'a self) -> u64set::Iter64<'a, T> {
+        self.x.iter()
+    }
+}
+
 impl<T: fmt::Debug + u64set::Fits64 + Ord> fmt::Debug for tinyset_Set64<T> {
```

Comment:
Does this mean the elements have to implement `Ord` for debugging? Is this too stringent?

Reply:
Same for Display

Reply:
Ah I see, you are requiring `Ord` for using sets at all time. Maybe we can also provide an "Unordered Set" type for those who really don't want / can't implement `Ord`?

Reply:
Thanks, I removed redundant trait bounds thoughout the file